### PR TITLE
Feat: Column level profiler dashboard 

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I have worked on adding Column level profiler dashboard
The unit test will be added in the next PR as this feature is still works in progress!

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/71748675/185220490-27da865c-dd1d-4909-a0a2-26ac2df7ae85.mov

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/axiosAPIs/tableAPI.ts [hunk 1]
```diff
@@ -212,3 +215,27 @@ export const putTableProfileConfig = async (
...
+  let url = `/tables/${tableId}/tableProfile`;
+  if (!isEmpty(params)) {
+    const urlParams = new URLSearchParams(params as Record<string, string>);
+    url = `${url}?${urlParams.toString()}`;
+  }
+
+  const response = await APIClient.get<{
+    data: TableProfile[];
+    paging: Paging;
+  }>(url);
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/axiosAPIs/tableAPI.ts:238]:
```suggestion
  let url = `/tables/${tableId}/tableProfile`;
 
  const response = await APIClient.get<{
    data: TableProfile[];
    paging: Paging;
  }>(URL,{params});
```

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+            tagsHandler={onTagUpdate}
+            tier={tier}
+            titleLinks={breadcrumb}
+            updateOwner={onOwnerUpdate}
+            updateTier={onTierUpdate}
+          />
+        </Col>
+        <Col span={24}>
+          <Row justify="space-between">
+            <Radio.Group
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx:315]: Can we use antd tab component here?
https://ant.design/components/tabs/

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+        owner: {
+          ...table.owner,
+          ...newOwner,
+        },
+      };
+      onTableChange(updatedTableDetails);
+    }
+  };
+
+  const onTierUpdate = (newTier?: string) => {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx:176]:
```suggestion
  const handleTierUpdate = (newTier?: string) => {
```

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+    } else {
+      return Promise.reject();
+    }
+  };
+
+  /**
+   * Formulates updated tags and updates table entity data for API call
+   * @param selectedTags
+   */
+  const onTagUpdate = (selectedTags?: Array<EntityTags>) => {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx:203]:
```suggestion
  const handleTagUpdate = (selectedTags?: Array<EntityTags>) => {
```

File: openmetadata-ui/src/main/resources/ui/src/axiosAPIs/tableAPI.ts (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/axiosAPIs/tableAPI.ts:238]: done

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+      const updatedTags = [...(tier ? [tier] : []), ...selectedTags];
+      const updatedTable = { ...table, tags: updatedTags };
+      onTableChange(updatedTable);
+    }
+  };
+
+  const unfollowTable = async () => {
+    try {
+      const data = await removeFollower(table.id, getCurrentUserId());
+      const { oldValue } = data.changeDescription.fieldsDeleted[0];
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx:214]: Can we get the current user and use the id from that and not use `getCurrentUserId` method?

File: openmetadata-ui/src/main/resources/ui/src/constants/entity.constants.ts [hunk 6]
```diff
@@ -47,6 +48,105 @@ export const PROFILER_METRIC = [
   'customMetricsProfile',
 ];
 
+export const PROFILER_FILTER_RANGE = {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/constants/entity.constants.ts:51]: Can we move all profiler-related constants to the profiler constant file?

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx:315]: I ll handle this in my next PR as this feature is still work in progress

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx (hunk 5 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx:214]: done

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**